### DISCUSSION TITLE
use-scroll-into-view does not work with ScrollArea component

### DISCUSSION THREAD

**Discussion by daler-sz** [mantinedev/mantine#5558] [Q&A] [answered]
Is that okay that `use-scroll-into-view` hook does not work if `scrollableRef` is `ScrollArea` component or I should create an issue?
If I replace `ScrollArea` with `Paper` component with `overflow: scroll`, it works properly
I use 7.4.1.
[Minimal reproduction](https://codesandbox.io/p/sandbox/great-wescoff-kppg82?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clrawbvzi0006356i5w4fum9g%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clrawbvzi0002356it1nz6w2i%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clrawbvzi0003356itf2g99qn%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clrawbvzi0005356ip6kdu7go%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clrawbvzi0002356it1nz6w2i%2522%253A%257B%2522id%2522%253A%2522clrawbvzi0002356it1nz6w2i%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clrawd5nd005c356ibu1y4f0n%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A6%252C%2522startColumn%2522%253A27%252C%2522endLineNumber%2522%253A6%252C%2522endColumn%2522%253A27%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252FApp.tsx%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clrawd5nd005c356ibu1y4f0n%2522%257D%252C%2522clrawbvzi0005356ip6kdu7go%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clrawbvzi0004356idmc74ayb%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522id%2522%253A%2522clrawbvzi0005356ip6kdu7go%2522%252C%2522activeTabId%2522%253A%2522clrawbvzi0004356idmc74ayb%2522%257D%252C%2522clrawbvzi0003356itf2g99qn%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clrawbvzi0003356itf2g99qn%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

**Comment by yverby** [comment, marked as answer]
Use `viewportRef` instead of `ref` prop
```jsx
<ScrollArea viewportRef={...} />
```

**Reply by daler-sz** [reply]
thanks, it worked
